Topics/Software_Engineering/Sprint Planning Meeting.md

```diff
@@ -0,0 +1,40 @@
+# Sprint Planning Meeting
+
+## Introduction
+In the Scrum framework, sprints function as short, consistent cycles during which a set amount of work is completed. Since they are the heartbeat of the framework, ensuring sprints are well planned, or ensuring a successful Sprint Planning Meeting, is a crucial part of the framework.
+
+
+## What does a Sprint Planning Meeting Encompass? 
+
+| Scrum Artifacts | Definition |
+| -------- | ------- |
+| Product Backlog  | an ordered list of tasks essential for improving the product and satisfying the stakeholders  |
+| Sprint Backlog | a list of tasks selected from the Product Backlog to be completed during a specific sprint     |
+
+* The meeting is held with the entire Agile team (including the ScrumMaster and Product Owner) and aims to conclude with a set of agreed upon **Product Backlog** items to be completed for the current sprint commitment. 
+* The ScrumMaster is responsible for leading the Planning Meeting and the Product Owner is responsible for detailing the Product Backlog items and ensuring alignment with the product’s goals. 
+* The planning addresses the value the sprint can bring to the stakeholders and the team collaborates to define a **Sprint Goal** before the end of the Sprint Planning meeting. 
+* Upon discussion with the Product Owner, the developers select items from the Product Backlog to include in the **Sprint Backlog** and determine this based on their prior and future Sprint performance. 
+* For each Sprint Backlog item that is selected, the developers on the team decide how to break down and complete tasks for the sprint based on the team’s definition of “Done”.
+
+
+## Integrating Sprint Planning Meetings in CSC301 
+*Consider an example of how a Scrum Planning Meeting can be integrated into our course’s workflow.*
+
+### Before the Planning Meeting:
+After a Product Manager and Scrum Master are chosen in Deliverable 1, consider each deliverable (deliverable 2 to 5) equivalent to a sprint, with each sprint having one dedicated Sprint Planning Meeting. As a ScrumMaster and Product Manager, organize an hour long meeting within the next week after the Deliverable handout is released. Prior to the Scrum Planning Meeting, you both should also read through the handout and identify features/Product Backlog items for the upcoming sprint and add them to the Product Backlog list on Jira (or the team’s chosen product management tool).
+
+### During the Planning Meeting: 
```

Comment:
Perhaps, you can more explicitly state how this helps on large projects.

Reply:
Thank you for the suggestion! I will add this in the _Introduction_.